starter-code/server.js

```diff
@@ -35,8 +38,12 @@ app.get('/articles', function(request, response) {
 
 app.post('/articles', function(request, response) {
   client.query(
-    '', // TODO: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
-    [], // TODO: Add the author and "authorUrl" as data for the SQL query
+    `INSERT INTO authors (author, "authorUrl")
+     VALUES ($1, $2) ON CONFLICT DO NOTHING;`, // Done! TODO: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
+    [
+      request.body.author,
+      request.body.authorURL
```

Comment:
this will cause issues. DO we have anything called authorURL?(HInt: Look at the spelling)